config/initializers/barong_domain.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Barong
+  class Config
+    class << self
+      def domain
+        app_domain = Barong::App.config.domain
```

Comment:
Would be cool to have `Barong::App.config.url`

Reply:
I can't override the `Barong::App.config.url` as a method. So, I will use `Barong::App.url` instead.